### ISSUE TITLE
Style tags populated with [object Object]

### ISSUE THREAD

**Issue by hairmot** [cheton/browserify-css#53]
This seems to have happened since the move from clean-css 2.2x to 4.1.5

I'm seeing my styles inserted as:

`<style type="text/css" data-href="src\shared\css\fancyLoadingButton.css">[object Object]</style>`

My settings in package.json are 

```
"browserify-css": {
    "autoInject": true,
    "verbose": true,
    "minify":true,
    "rootDir": "."
  }
```

if i remove the minify:true from package json, my styles are included correctly.

Problem is the same if i put my config in a separate browserify-css,js file

Has anyone else had any issues with a similar config?

**Comment by Melatonin64** [comment]
I also encountered this issue, switched back to previous version for now.

**Comment by cheton** [comment]
@hairmot @Melatonin64 

Thank you for reporting and fixing this issue. The bug fix is now available with v0.11.1.
